src/aap_eda/settings/post_load.py

```diff
@@ -437,6 +437,16 @@ def post_loading(loaded_settings: Dynaconf):
             "process_manager_kwargs": {
                 "preload_modules": ["aap_eda.core.dispatcherd_pre_fork"],
             },
+            "min_workers": settings.get(
+                "DISPATCHERD_MIN_WORKERS", settings.DISPATCHERD_MIN_WORKERS
+            ),
+            "max_workers": settings.get(
+                "DISPATCHERD_MAX_WORKERS", settings.DISPATCHERD_MAX_WORKERS
+            ),
+            "target_workers": settings.get(
```

Comment:
I don't think target_workers is a thing? That would just be like the same as min_workers?

Reply:
@Alex-Izquierdo the amount of CPU cores is 4, which raised the warning message now.

Reply:
> I wonder if by default we should use the default value and maybe override it on-demand to tune it up for more specific scenarios. Are you sure the node is not just running too many activations in this usecase if it only has 4 cores?

Not sure, what are your suggestions in this case? @Alex-Izquierdo @AlanCoding

Reply:
> I wonder if by default we should use the default value and maybe override it on-demand to tune it up for more specific scenarios.

Sounds like you're describing the min_workers (idle number) vs max_workers. Having a low min_workers can increase latency to deal with certain things, but hitting max_workers is often death for tests or user experience.

Reply:
Oh dear, we really were on defaults here

https://github.com/ansible/dispatcherd/blob/809218a891d8ef0d0cfbc62b9e3cbfdc735a6b68/dispatcherd/service/pool.py#L210

that's 1 for `min_workers` and then the CPU count for `max_workers`.